docs(agents|workflows): Update litmusctl usage docs

PrayagS opened the pull request:
**What this PR does / why we need it**:

This PR adds the following changes in litmusctl documentation,
- Refactor `create agent` to `connect agent`.
- Add `REGISTRATION` field in `get agents` output.
- Add `disconnect agent` command under additional commands.
- Add instructions for `create workflow` command.
- Add other workflow commands viz. get, delete, and describe under additional commands.

**Checklist:**
-   [ ] Fixes #<issue number>
-   [x] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

Signed-off-by: PrayagS <prayagsavsani@gmail.com>

shubhamchaudhary commented on docs/openebs-pool-pod-failure.md:
```diff
@@ -136,32 +185,33 @@ metadata:
-            value: 'pvc-c466262a-a5f2-4f0f-b594-5daddfc2e29d'    
-          - name: DEPLOY_TYPE
-            value: deployment        
+          env:
```
@ksatchit We have `components.env` field here But this field is not present in our 1.0.0 release. It is only present in ci image. Is it okay to put this change here?

ksatchit replied:
this is the intention of putting it into master :)  -- this will go into docs master ..so 'next' in URL

shubhamchaudhary replied:
okay got it, Thanks

PrayagS committed:
docs(agents): Update docs on agent commands

- Refactor `create agent` -> `connect agent`.
- Add `REGISTRATION` field in `get agents`.
- Add `disconnect agent` under additional commands.

Signed-off-by: PrayagS <prayagsavsani@gmail.com>

PrayagS committed:
docs(workflows): Add docs on workflow commands

Signed-off-by: PrayagS <prayagsavsani@gmail.com>
